packages/user-interface/src/components/formio/useFormIoState.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+
+export type FormIoRefState = {
+  //eslint-disable-next-line @typescript-eslint/no-explicit-any
+  setState: ({ value }: { value: any }) => void;
```

Comment:
Kunnen we voor any ook unknown gebruiken? Daar geeft hij misschien geen errors op?